lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1130,9 +1141,16 @@ instance Arbitrary ApiWalletMigrationInfo where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
-instance Arbitrary (ApiWalletMigrationPostData n) where
+instance Arbitrary (ApiWalletMigrationPostData n "lenient") where
     arbitrary = do
-        n <- choose (1,256)
+        n <- choose (1,255)
+        pwd <- arbitrary
+        addr <- vector n
+        pure $ ApiWalletMigrationPostData pwd ((, Proxy @n) <$> addr)
+
+instance Arbitrary (ApiWalletMigrationPostData n "raw") where
+    arbitrary = do
+        n <- choose (1,255)
         pwd <- arbitrary
         addr <- vector n
         pure $ ApiWalletMigrationPostData pwd ((, Proxy @n) <$> addr)
```

Comment:
Perhaps we can reduce code duplication here by extracting out a common function and using it from both instances?

Reply:
Fixed